protx-client/src/components/ProTx/components/charts/MainPlot.css

```diff
@@ -15,4 +15,6 @@
 }
 .main-plot {
   margin: 0 1vw;
+  width: 100%;
```

Comment:
This should be restricted to `width: 96%;`. 
- If it is above 96%, it generates the horizontal scrollbar on the iframe element.
-- We should be able to improve this when I get the new iframe template PR in.
- If it is above 98%, it cuts off the text labels on maltreatment plot on the right-side.

Given the minute difference between 96 and 98%, I recommend the 96% to eliminate the horizontal scrolling issue and clean up the entire page layout (again, pending a better template and unified way to handle iframe embedding).

Reply:
```suggestion
  width: 96%;
```